src/List.test.jsx

```diff
@@ -0,0 +1,50 @@
+import { fireEvent, render } from '@testing-library/react';
+
+import List from './List';
+
+describe('Item', () => {
+  const onClickDelete = jest.fn();
+
+  function renderList(tasks) {
+    return render(
+      <List taskTitle="" tasks={tasks} onClickDelete={onClickDelete} />,
+    );
```

Comment:
```suggestion
    return render((
      <List 
        taskTitle=""
        tasks={tasks}
        onClickDelete={onClickDelete}
      />
    ));
```

저는 속성이 2개 이상일 때는 줄바꿈해주면 코드가독성이 더 좋다고 생각하고 코드를 정리해주는 편이에요.
코드 가독성을 고려한 자기만의 코드 정리 기준을 세우시면 좋아요.